src/app/services/search/search.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class SearchService {
+
+  apiUrl = "https://landville-backend-web-api.herokuapp.com/api/v1/properties/?";
+
+  constructor(private http: HttpClient) { }
+
+  searchProperties(search: string) {
+    let searchUrl = this.apiUrl + search;
```

Comment:
In case you decide to go with my suggestion above, you can then go ahead to make the required change here:
```javascript
let searchUrl = `/properties/?${search}`;   // instead of concatenating with +

return this.http.getRequestWithParams(searchUrl);
```

(I don't know if there's a convention in the team to keep the code ES6 compliant. If there is, then you can use the template literal syntax instead of concatenating the string with a +)